homeassistant/components/zha/core/device.py

```diff
@@ -99,8 +100,13 @@ def __init__(
             self._zigpy_device.__class__.__module__,
             self._zigpy_device.__class__.__name__,
         )
+        self._consider_unavailable_time = (
```

Comment:
We prefer writing regular if else blocks if the ternary operator spans more than one line.

Reply:
is there a reason? That seems kind of arbitrary. I mean I could just use bad 2 character variable names and keep it on 1 line. I don’t have a problem changing this at all... just playing devils advocate with the preference since in cases like this the code is no more or less clear in either fashion.

Reply:
It's just our convention.

Reply:
Fair enough!

Reply:
updated